apps/couchbase.go

```diff
@@ -308,7 +309,7 @@ func (lp LoggingProcessorCouchbaseHTTPAccess) Components(tag string) []fluentbit
 	c := lp.LoggingReceiverFilesMixin.Components(tag)
 	c = append(c,
 		confgenerator.LoggingProcessorParseRegex{
-			Regex: `^(?<client_ip>[^ ]*) [^ ]* (?<user>[^ ]*) \[(?<timestamp>[^\]]*)\] "(?<method>\S+)(?: +(?<path>[^ ]*) +\S*)?" (?<status_code>[^ ]*) (?<response_size>[^ ]*) - (?<message>.*)$`,
+			Regex: `^(?<http_request_remoteIp>[^ ]*) (?<host>[^ ]*) (?<user>[^ ]*) \[(?<timestamp>[^\]]*)\] "(?<http_request_requestMethod>\S+)(?: +(?<http_request_requestUrl>[^ ]*) +\S*)?" (?<http_request_status>[^ ]*) (?<http_request_responseSize>[^ ]*) - "(?<http_request_userAgent>[^\"]*)")(?<message>.*)$`,
```

Comment:
```suggestion
			Regex: `^(?<http_request_remoteIp>[^ ]*) (?<host>[^ ]*) (?<user>[^ ]*) \[(?<timestamp>[^\]]*)\] "(?<http_request_requestMethod>\S+)(?: +(?<http_request_requestUrl>[^ ]*) +\S*)?" (?<http_request_status>[^ ]*) (?<http_request_responseSize>[^ ]*) - "(?<http_request_userAgent>[^\"]*)"(?<message>.*)$`,
```

Was noticing this as an extra parentheses in regex101, I think it tracks, but I'm spinning up an instance just to validate real quick, so don't accept this change quite yet!